docs/articles/multi-attribute-semantic-search.md

```diff
@@ -0,0 +1,459 @@
+    ...
+  ],
+  "queries": [
+    {"look": "...", "habitat": "...", "behavior": "..."},
+    ...
+  ]
+}
+```
+
+Let's examine a sample of our generated dataset. Note that your results may differ due to the non-deterministic nature of LLM generation. Original dataset is available [here](https://github.com/superlinked/article-multi-attribute-search).
+
+First five monsters:
+
+
+|    | name             | look                                                      | habitat                                             | behavior                                                      |
+|---:|:-----------------|:----------------------------------------------------------|:----------------------------------------------------|:--------------------------------------------------------------|
+|  0 | Luminoth         | Moth-like creature with glowing wings and antenna         | Dense forests and jungles with bioluminescent flora | Emits soothing light patterns to communicate and attract prey |
+|  1 | Aqua Wraith      | Translucent humanoid figure made of flowing water         | Rivers, lakes, and coastal areas                    | Shapeshifts to blend with water bodies and controls currents  |
+|  2 | Stoneheart Golem | Massive humanoid composed of interlocking rock formations | Rocky mountains and ancient ruins                   | Hibernates for centuries, awakens to protect its territory    |
+|  3 | Whispering Shade | Shadowy, amorphous being with glowing eyes                | Dark forests and abandoned buildings                | Feeds on fear and whispers unsettling truths                  |
+|  4 | Zephyr Dancer    | Graceful avian creature with iridescent feathers          | High mountain peaks and wind-swept plains           | Creates mesmerizing aerial displays to attract mates          |
+
+And here are the queries:
+
+|    | look          | habitat              | behavior              |
+|---:|:--------------|:---------------------|:----------------------|
+|  0 | glowing       | dark places          | light manipulation    |
+|  1 | elemental     | extreme environments | environmental control |
+|  2 | shapeshifting | varied landscapes    | illusion creation     |
+|  3 | crystalline   | mineral-rich areas   | energy absorption     |
+|  4 | ethereal      | atmospheric          | mind influence        |
+
+## Retrieval
+
+Let's set up common parameters for both approaches we'll explore: the naive method and the Superlinked method.
+
+We'll use the `sentence-transformers/all-mpnet-base-v2` model from the Sentence Transformers library to generate our vector embeddings. To keep our results short, we'll limit the output to the top 3 matches.
+
+```python
+LIMIT = 3
+MODEL_NAME = "sentence-transformers/all-mpnet-base-v2"
+```
+
+For the complete code, including necessary imports and helper functions, please refer to the [accompanying notebook](https://github.com/superlinked/article-multi-attribute-search/blob/main/notebook.ipynb).
```

Comment:
ohhh I see, you created a github repo. You should just make it public then I suppose.